### PR TITLE
Modernize things

### PR DESCRIPTION
**Pull request by Goooler**

### CODE REVIEW

File: src/test/fixtures/settings.gradle.kts
Comment (Goooler) [inline src/test/fixtures/settings.gradle.kts:1]: Reuse this file for most of fixture projects, copy it by:
```kotlin
  private fun reuseSettingsGradle(fixtureDir: File) {
    if (
      !File(fixtureDir, "settings.gradle.kts").exists() &&
      !File(fixtureDir, "settings.gradle").exists()
    ) {
      File(fixtureDir.parentFile, "settings.gradle.kts").copyTo(File(fixtureDir, "settings.gradle.kts"), true)
    }
  }
```
Not sure if there is any better way to reuse the same logic for the others.

**Comment by hfhbd** [comment]
I didn't see your PR..., I did a similar thing in #228

File: src/test/fixtures/settings.gradle.kts
Comment (hfhbd) [reply src/test/fixtures/settings.gradle.kts:1]: I used an included build with a custom settings plugin and I applied it in each build.

Comment (Goooler) [reply src/test/fixtures/settings.gradle.kts:1]: Good to see!

**Comment by Goooler** [comment]
Just reverted 3a791a93c6467cd21000670802c227b405284d4f & a97a6dd4f1ceb9c4a4c218418a1d6d6b781c0d2c, let's merge your PR first!

**Pull request merged**